wireapp/wire-ios-mono
# feat: redefine contact list [WPB-3632]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-3632" title="WPB-3632" target="_blank"><img alt="Topic" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/15209?size=medium" />WPB-3632</a>  Redefine contact list / Remove team member updates
  </td></table>
  <br />
 




















----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

This PR merges a feature branch with changes of three tickets related to team events not being sent anymore:

[WPB-4538](https://wearezeta.atlassian.net/browse/WPB-4538) Stop processing team events
[WPB-4540](https://wearezeta.atlassian.net/browse/WPB-4540) Rely on remote search when searching for users
[WPB-4541](https://wearezeta.atlassian.net/browse/WPB-4541) Stop maintaining a local list of all team members

### Testing

#### How to Test

Connect the app to a testing environment and ensure, that your team members can be found using the search.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.


[WPB-4538]: https://wearezeta.atlassian.net/browse/WPB-4538?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
